packages/client/src/views/Settings/SettingsPage.tsx

```diff
@@ -112,7 +112,7 @@ const TopAlignedListViewItemSimplified = styled(ListViewItemSimplified)`
   align-items: start;
   padding: 16px 0;
 `
-const DynamicHeightLinkButton = styled(LinkButton)`
+export const DynamicHeightLinkButton = styled(LinkButton)`
```

Comment:
Could you move this component into its own module that both `SettingsPage` and `AvatarChangeModal` would use as a dependency? Otherwise, there's a cyclic dependency where `SettingsPage` depends on `AvatarChangeModal` that again depends on `SettingsPage`